api.js

```diff
@@ -56,7 +61,15 @@ Api.prototype._runFile = function (file) {
 		args.push('--sorted');
 	}
 
-	return fork(args)
+	var options = {};
+
+	if (Array.isArray(this.require) && this.require.length) {
```

Comment:
Just do 
```js
if (this.require) {
 // ...
}
```

You have already guaranteed it is an array of at least length one above.